packages/bahmni-widgets/src/documentPrintButton/DocumentPrintButton.tsx

```
@@ -16,77 +30,129 @@ interface DocumentPrintButtonProps {
+      <InlineLoading description={descriptionText} />
+    );
   }
 
-  const handlePrint = (option: PrintOption) => {
-    triggerPrint(option.templateId);
-  };
+  if (iconOnly) {
+    if (items.length === 1) {
+      return (
+        <IconButton
+          label={ t(items[0].translationKey) ?? iconLabel}
```

ESLint

[error] There should be no space after '{'

(react/jsx-curly-spacing)